test/unit/physics/utils/test_orbital_data_stubs.F90

```diff
@@ -0,0 +1,50 @@
+! Copyright (C) 2024 National Science Foundation-National Center for Atmospheric Research
+! SPDX-License-Identifier: Apache-2.0
+!--------------------------------------------------------------------------
+!
+! Stub modules for the orbital_data module tests.
+!
+!--------------------------------------------------------------------------
+
+module shr_kind_mod
+  implicit none
+  integer, parameter :: SHR_KIND_R8 = selected_real_kind(15)
+end module shr_kind_mod
+module shr_const_mod
+  use shr_kind_mod, only: R8 => SHR_KIND_R8
+  implicit none
+  real(kind=R8), parameter :: SHR_CONST_PI = 3.14159265358979323846_R8
```

Comment:
Is `SHR_CONST_PI` needed here?  It doesn't look like it is being used anywhere (but I could just be missing something).

Reply:
good catch, I've removed it